### ISSUE TITLE
Unused smartgroups

### ISSUE THREAD

**Issue by rossoneris** [BIG-RAT/Prune#11]
This is not a issue - but rather a request/question

I have several smartgroups that are empty because clients does not match the smartgroup - but if certain things change on the client, they then can get part of this smartgroup

Is there a way to filter smartgroups that are not setup in any policy ? - as If smartgroups are not used in a policy, then for sure it is obsolete. Just to remove empty smart groups can give issues, if they are part of certain policies and maybe now and then a client will drop into this smart group

**Comment by BIG-RAT** [comment]
This hits on an item that's bothered my for a while.  Not being able to identify smart groups that are not linked to any policies and are only used to send an email when it changes.  

Do like the idea.  When I get around to adding some preference settings for parameters around the searches that would be a good one, along with last device check-in, last time a policy ran....

**Comment by rossoneris** [comment]
Any update on my wishes ? :)

**Comment by BIG-RAT** [comment]
I should have asked, are empty smart groups used in a policy/config profile being listed?  They shouldn't be.

**Comment by rossoneris** [comment]
Yes - use lots of them. If Applications/processes are not running, put them in X smart group, so it will correct an error

**Comment by BIG-RAT** [comment]
And those smart groups (that are empty) show as unused even though they are used for scoping a policy, config profile, or as criteria for another group?
